services/app-web/src/pages/StateSubmission/ReviewSubmit/ReviewSubmit.test.tsx

```diff
@@ -238,4 +240,221 @@ describe('ReviewSubmit', () => {
 
         expect(errorText).toBeInTheDocument()
     })
+
+    it('opens submission modal with summary input when submit button is clicked on unlocked plan package', async () => {
```

Comment:
These tests look really good 🏆 can you wrap them in a jest `describe` block because I feel like they are all related and also anticipating we are going to keep adding more and more complex logic to this page